operator/ccnp_event.go

```diff
@@ -39,7 +40,11 @@ import (
 // using CiliumNetworkPolicy itself, the entire implementation uses the methods
 // associcated with CiliumNetworkPolicy.
 func enableCCNPWatcher() error {
-	log.Info("Starting to garbage collect stale CiliumClusterwideNetworkPolicy status field entries...")
+	enableCNPStatusUpdates := kvstoreEnabled() && option.Config.K8sEventHandover && !option.Config.DisableCNPStatusUpdates
+	if enableCNPStatusUpdates {
+		log.Info("Starting a CCNP Status handover from etcd to k8s...")
```

Comment:
Theoretically, we could have other kvstores like Consul, so maybe changing this log msg to say "kvstore" instead makes sense?